test/whitelisted_crowdsale.js

```diff
@@ -0,0 +1,75 @@
+"use strict";
+
+const MFC_Token = artifacts.require("MFC_Token");
+const WhitelistedCrowdsale = artifacts.require("WhitelistedCrowdsale");
+
+contract('WhitelistedCrowdsale', async (accounts) => {
+  let owner = accounts[0];
+  let user = accounts[1];
+
+  let now = Date.now();
+  let twoDays = 1000 * 60 * 60 * 24 * 2;
+
+  let startTime = now - twoDays;
+  let endTime = now + twoDays;
+  let rate = 30;
+  let minPurchase = 1;
+  let maxPurchase = 1000;
+  let wallet = user;
+  let countPurchaseAmount = false;
+  
+  let token = null;
+  let instanceWhitelistedCrowdsale = null;
+
+  beforeEach("setup contract for each test", async () => {
+    token = await MFC_Token.new({from: owner});
+    instanceWhitelistedCrowdsale = await WhitelistedCrowdsale.new(
```

Comment:
@avgoeid I thought you've decided that the test should use `ICO_controller.startPrivateOffer` instead of direct constructor, no?

Reply:
Maybe it will be better extend ICO_controller test case?

Reply:
@avgoeid maybe, what I am saying is that when you discussed this with @Norfolks , I had an impression that you will use ICO_controller.startPrivateOffer.
Maybe we can keep this text and add another task for `ICO_controller.startPrivateOffer`, please talk about this with @Norfolks once more.